components/quizzes/SkillifyNavbar.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+type SkillifyNavbarProps = {
+  hidden: boolean;
+  onBackClick: () => void;
+};
+const SkillifyNavbar = ({ hidden, onBackClick }: SkillifyNavbarProps) => {
+  const className = hidden ? "w-16 p-4 opacity-0" : "w-16 p-4";
+  return (
+    <div className="flex justify-between">
+      <div className="flex justify-start">
+        <img
+          src="/images/backarrow.svg"
+          onClick={onBackClick}
```

Comment:
onBackClick handler decrements the stages state variable and navigates the user backwards